plugins/SitesManager/templates/_vueTabInstructions.twig

```diff
@@ -1,3 +1,12 @@
+<p></p><p></p>
```

Comment:
Seems a bit hacky, is there a better way of doing this?

Reply:
Sure. But I only moved that code around, didn't want to apply changes to the html structure, as that would have caused changes in the UI tests as well.